stories/atoms/components/button/Button.stories.mdx

```diff
@@ -174,3 +186,38 @@ The `Button` component can be disabled at any time, thus preventing any user int
     )}
   </Story>
 </Canvas>
+
+## Content
+
+The content can be customized to fit users needs through its properties `label`, `icon`,
+`leftIcon` and `rightIcon`.
+
+- `label` is the only obligatory property and is used to clearly indicate what action will be triggered, through the
+  tooltip and text.
+- `icon` sets the icon for the **variant** 'icon'.
+- `leftIcon` and `rightIcon` are auxilliary elements giving great freedom to embellish the button
+  together with the label.
+
+<Canvas>
+  <Story name="Content">
+    {args => (
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+          gap: '25px',
+          flexWrap: 'wrap'
+        }}
+      >
+        <Button icon={<Icon name="wallet" />} label="wallet" variant="icon" />
+        <Button leftIcon={<Icon name="wallet" />} label="Get wallet address" />
```

Comment:
Could you decrease a bit the icon sizes please? (30px looks too big in the button)

Reply:
How about if I decrease it to 25px? I find that if I decrease it too much the details of the icon become too small.